project/Dependencies.scala

```diff
@@ -11,8 +11,11 @@ object Dependencies {
   }
 
   val AwsSdkVersion = "1.11.476"
+  val AwsSdk2Version = "2.3.9"
   val AkkaHttpVersion = "10.1.7"
-  
+
+  val Java8CompatVersion = "0.9.0"
```

Comment:
A short heads-up on this:
Can you get around upgrading? We do not want to override Akka's version of this dependency.

Reply:
I'll fix this today.

Reply:
Done